Entities/User.java

```diff
@@ -0,0 +1,44 @@
+package io.github.djunicode.djcomps.Entities;
+
+import android.arch.persistence.room.Entity;
+import android.arch.persistence.room.ForeignKey;
+import android.arch.persistence.room.PrimaryKey;
+
+import java.math.BigInteger;
+
+import static android.arch.persistence.room.ForeignKey.CASCADE;
+
+/**
+ * Created by Jai on 30-12-2017.
+ */
+
+@Entity(foreignKeys = @ForeignKey(entity = Group.class,
+        parentColumns = {"group_id", "student", "year", "division"},
+        childColumns = {"group_id", "student", "year", "division"},
+        onDelete = CASCADE))
+
+
+public class User {
+    @PrimaryKey public BigInteger sapid;
+    public char password;

```

Comment:
This is just for offline data.
We don't have access to other users password and diskUtilization